src/buttons/Button/Button.jsx

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import { useState } from 'preact/hooks';
+
+import { MopIcon } from '@lana/b2c-mapp-ui-assets';
+import CSS from './styles.css';
+
+const validTypes = ['secondary', 'disabled', 'dismiss'];
+
+const Button = ({ children, className, dataTestId, href, id, loading, type, onClick }) => {
+	const [isPressed, setIsPressed] = useState(false);
+
+	const onHandleClick = event => {
+		if (onClick) {
+			onClick(event);
+		}
+		if (isPressed) {
+			setIsPressed(!isPressed);
+		}
+		event.preventDefault();
+	};
+
+	const typeClass = validTypes.includes(type) ? CSS[type] : '';
+	const loadingClass = loading ? CSS.loading : '';
+	const pressedClass = isPressed ? CSS.pressed : '';
+	const isButton = !href;
+	const defaultTestId = isButton ? 'button' : 'button-link';
+	const testId = dataTestId || defaultTestId;
+
+	if (isButton) {
+		return (
+			<button 
+				data-testid={`${testId}-button`} 
+				type="button" id={id} 
+				onTouchStart={() => setIsPressed(!isPressed)} 
```

Comment:
(DRY) please refactor this arrow-function into a new `const` above line: 22 and then reuse it here and everywhere else you have `() => setIsPressed(!isPressed)`